File: Quiz-server/src/main/java/com/qucat/quiz/repositories/entities/Activity.java
package com.qucat.quiz.repositories.entities;

import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.Date;

@Data
@AllArgsConstructor
public class Activity {
    private int id;
    private String friendLogin;
    private String achievementName;
    private String quizName;
    private String quizCategoryName;
    private boolean markedFavourite;
    private String friendOfFriendLogin;
    private Date activityDate;
    private int imageId;
    private Image image;
}

File: Quiz-web/src/app/modules/core/models/activity.ts
import {Image} from "./image";
import {Imaged} from "./imaged";

export class Activity implements Imaged{
  id: number;
  friendId:number
  friendLogin:string;
  achievementName:string;
  quizId:number;
  quizName:string;
  quizCategoryName:string;
  markedFavourite:boolean;
  friendOfFriendId:number;
  friendOfFriendLogin:string;
  activityDate: Date;
  imageId: number = -1;
  image: Image;
}
